### PR TITLE
Change grpc to http

### PR DESCRIPTION
**Pull request by omnipresent07**
This PR addresses the memory leak issue with GRPCs

### CODE REVIEW

File: kubernetes/hmda-platform/values.yaml [hunk 1]
```diff
@@ -81,6 +81,14 @@ grpc:
     port: 60081
     host: census-api-grpc.default.svc.cluster.local
 
+http:
+  check_digit:
+    port: 9091
+    host: check-digit-http.default.svc.cluster.local
```
Comment (kgudel) [inline kubernetes/hmda-platform/values.yaml]: this should be: `check-digit.default.svc.cluster.local`

File: census-api/src/main/scala/hmda/census/validation/CensusValidation.scala [hunk 2]
```diff
@@ -9,6 +9,10 @@ object CensusValidation {
     indexedTract.contains(tract)
   }
 
+  def isTractValidTest(tract: String): Boolean = {
```
Comment (PatrickHSI) [inline census-api/src/main/scala/hmda/census/validation/CensusValidation.scala]: @omnipresent07 does this need to be implemented? Looks like it's just a placeholder to default to true

File: check-digit/src/main/scala/hmda/uli/HmdaUli.scala [hunk 3]
```diff
@@ -26,5 +26,5 @@ object HmdaUli extends App {
   implicit val system = ActorSystem("hmda-uli")
 
   system.actorOf(HmdaUliApi.props(), "hmda-uli-api")
-  system.actorOf(HmdaUliGrpc.props(), name = "hmda-uli-grpc")
+//  system.actorOf(HmdaUliGrpc.props(), name = "hmda-uli-grpc")
```
Comment (PatrickHSI) [inline check-digit/src/main/scala/hmda/uli/HmdaUli.scala]: If you're removing GRPC you can go ahead remove this commented out code. Doesn't look like you're leaving any other references to GRPC in place for roll back.

Comment (omnipresent07) [reply check-digit/src/main/scala/hmda/uli/HmdaUli.scala]: New commits fix the review comments.

**Pull request merged**